common/taskana-common/src/main/resources/sql/postgres/taskana_schema_update_4.3.0_to_4.8.0_postgres.sql

```diff
@@ -5,3 +5,43 @@ SET search_path = %schemaName%;
 INSERT INTO TASKANA_SCHEMA_VERSION (VERSION, CREATED) VALUES ('4.8.0', CURRENT_TIMESTAMP);
 
 ALTER TABLE TASK ADD COLUMN RECEIVED TIMESTAMP NULL;
+ALTER TABLE TASK ADD COLUMN MANUAL_PRIORITY INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_1 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_2 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_3 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_4 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_5 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_6 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_7 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_8 INT NULL;
+
+ALTER TABLE CONFIGURATION ADD COLUMN CUSTOM_ATTRIBUTES CLOB NULL;
+
+ALTER TABLE SCHEDULED_JOB ALTER COLUMN TYPE SET DATA TYPE VARCHAR(255);
+
+ALTER TABLE CLASSIFICATION ALTER COLUMN DOMAIN SET DATA TYPE VARCHAR(32);
+
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN TASK_PRIORITY INT NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN TASK_PLANNED TIMESTAMP NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN TASK_DUE TIMESTAMP NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN TASK_OWNER VARCHAR(32) NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN TASK_CLASSIFICATION_NAME VARCHAR(255) NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN ATTACHMENT_CLASSIFICATION_NAME VARCHAR(255) NULL;
+ALTER TABLE TASK_HISTORY_EVENT ADD COLUMN WORKBASKET_NAME VARCHAR(255) NULL;
+
+CREATE TABLE USER (
```

Comment:
This throws a syntax error. USER is a key word on Postgres. Please read here: https://stackoverflow.com/questions/22256124/cannot-create-a-database-table-named-user-in-postgresql

Reply:
renamed to USER_INFO